partner_multi_relation/views/res_partner_relation_all.xml

```diff
@@ -123,27 +123,4 @@
         </field>
     </record>
 
-    <record
```

Comment:
What is the reason for deleting this action?

Reply:
It's a duplicate:
https://github.com/OCA/partner-contact/blob/12.0/partner_multi_relation/views/menu.xml#L6